certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/unit/model/UnitStatistics.java

```diff
@@ -0,0 +1,21 @@
+package se.inera.intyg.certificateservice.domain.unit.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Setter
+@Getter
+@EqualsAndHashCode
+@ToString
+public class UnitStatistics {
```

Comment:
Use a record or clean up the Lombok-annotations?

Reply:
I removed the ToString-annotation since it was only used to better understand why the tests were failing.